Find a functionally equivalent code:android.os.Bundle.putBooleanArray:COMMENT
Method Modifier: public      
Comment:/**
 * Inserts a boolean array value into the mapping of this Bundle, replacing
 * any existing value for the given key.  Either key or value may be null.
 *
 * @param key a String, or null
 * @param value a boolean array object, or null
 */

@@ -1,4 +1,3 @@
 {
-    unparcel();
-    mMap.put(key, value);
+    super.putBooleanArray(key, value);
 }

